CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-## 1.7.0 (Unreleased)
+## 1.6.2 (August 17, 2020)
+BUG FIXES:
+* `resource/resource_turbot_google_directory`: After creation of resource, when running `terraform plan`, `hosted_name` attribute gives a `diff`. ([#75](https://github.com/turbot/terraform-provider-turbot/issues/75))
```

Comment:
After creation of resource, `terraform plan` shows a change required  for attribute `hosted_name`. ([#75](https://github.com/turbot/terraform-provider-turbot/issues/75))